src/Sylius/Behat/spec/Context/Setup/LexicalContextSpec.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Behat\Context\Setup;
+
+use Behat\Behat\Context\Context;
+use PhpSpec\ObjectBehavior;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class LexicalContextSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith();
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Behat\Context\Setup\LexicalContext');
+    }
+
+    function it_implements_context_interface()
+    {
+        $this->shouldImplement(Context::class);
+    }
+
+    function it_transforms_price_string_to_integer()
+    {
+        $this->getPriceFromString('10.00')->shouldReturn(1000);
```

Comment:
It would be nice to have more cases covered as converting string to float and then float to int is quite unreliable.